src/functionalTest/java/uk/gov/hmcts/reform/blobrouter/FunctionalTestBase.java

```diff
@@ -31,7 +34,25 @@ protected void setUp() {
             config.sourceStorageAccountKey
         );
 
-        this.blobRouterStorageClient = new BlobServiceClientBuilder()
+        BlobServiceClientBuilder blobServiceClientBuilder = new BlobServiceClientBuilder();
+
+        if (Boolean.TRUE.equals(config.useProxyForSourceStorage)) {
```

Comment:
this won't work ? 

```Boolean.valueOf(config.storageProxyEnabled) == true```

Reply:
Probably. Just drop the `== true` bit if going to use

Reply:
why not just `if(!config.useProxyForSourceStorage)`?

Reply:
I thought it can be null. But seems conf does not support null value.